frontend/constants/urls.ts

```diff
@@ -14,6 +14,9 @@ const SWAP_BASE_URL: Url = 'https://balancer.fi/swap/base/ETH/OLAS';
 // TODO: Modius - confirm this URL is correct
 const SWAP_MODE_URL: Url =
   'https://jumper.exchange/?fromChain=34443&fromToken=0x0000000000000000000000000000000000000000&toChain=34443&toToken=0xcfD1D50ce23C46D3Cf6407487B2F8934e96DC8f9';
+// TODO: celo - confirm this URL is correct
+const SWAP_CELO_URL: Url =
+  'https://app.ubeswap.org/#/swap?inputCurrency=0x471ece3750da237f93b8e339c536989b8978a438&outputCurrency=0xacffae8e57ec6e394eb1b41939a8cf7892dbdc51';
```

Comment:
I think the url is correct https://olas.network/olas-token#get-olas
```suggestion
const SWAP_CELO_URL: Url =
  'https://app.ubeswap.org/#/swap?inputCurrency=0x471ece3750da237f93b8e339c536989b8978a438&outputCurrency=0xacffae8e57ec6e394eb1b41939a8cf7892dbdc51';
```